### PR TITLE
Updated the documentation to include a description of the archive metadata.json file

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,68 @@
...
+
+| Key         | Required | Description |
+|-------------|----------|-------------|
+| filename    | true     | The filename of the archive object. This should be unique within the manifest |
+| timestamp   | true     | The file creation date |
+| contentType | false    | The content type of the file |
+| jsonSchema  | false    | The uri for the json schema if the content type is 'application/json'. |
+
+Note: There are other fields in the manifest as described by the json schema, but these 
+are the fields that are most commonly asked about. syoung 09/27/2023
```
Comment (Erin-Mounts) [inline README.md]: https://xkcd.com/1179/
I don't actually feel all that strongly about it but it amused me to link this here

**Review by Erin-Mounts — APPROVED** [review]
One comment, looks good

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Review by DwayneJengSage — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**